docs/janssen-server/auth-server/endpoints/access-evaluation.md

```
@@ -38,9 +54,23 @@ https://janssen.server.host/jans-auth/.well-known/openid-configuration
 https://janssen.server.host/jans-auth/.well-known/authzen-configuration
 ```
 
-`/.well-known/authzen-configuration` allows to publish data specific to AuthZEN only. Response of AuthZEN discovery endpoint can be
+`/.well-known/authzen-configuration` allows publishing data specific to AuthZEN only. Response of AuthZEN discovery endpoint can be
 changed via `AccessEvaluationDiscoveryType` custom script.
 
+**Sample AuthZEN Discovery Response**
+```json
+{
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)